pixels-daemon/src/main/java/io/pixelsdb/pixels/daemon/monitor/util/PolicyManager.java

```diff
@@ -0,0 +1,40 @@
+package io.pixelsdb.pixels.daemon.monitor.util;
+
+import io.pixelsdb.pixels.common.utils.ConfigFactory;
+import io.pixelsdb.pixels.daemon.monitor.MetricsQueue;
+import io.pixelsdb.pixels.daemon.monitor.policy.BasicPolicy;
+import io.pixelsdb.pixels.daemon.monitor.policy.PidPolicy;
+import io.pixelsdb.pixels.daemon.monitor.policy.Policy;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class PolicyManager implements Runnable {
```

Comment:
What is this PolicyManager used for?

Reply:
Manage a background thread, check values in the queue periodically and do auto scaling according to the specified policy